### PR TITLE
ci(codeowners): proposal update for code owners

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

This PR is intended to update the scope of the CODEOWNER file and empower team manage their area of responsibility.

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+#patches/
+#locales/
+#docs/
+
+#Team contributors have ability to approve and merge PRs on specific directories
+#Swaps Team (all on-ramp team members)
+#On-Ramp Team (all on-ramp team members)
+#SDK Team (all SDK team members)
+#Accounts Team (all Accounts team members)
+#Confirmation Team (all Confirmation team members)
```
Comment (gantunesr) [inline .github/CODEOWNERS:21]: What would be the division for shared parts?

I'm wondering about the specific case Accounts and Confirmations regarding hardware wallet

Comment (sethkfman) [reply .github/CODEOWNERS:21]: I think we would just add both teams to that area of the code. @gantunesr

**Review by andreahaku — COMMENTED** [review]
I think you should rebase this branch as there are some missing folders like `app/core/WalletConnect` where now everything related to WC is hosted

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,30 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/mobile-devs
+*  @MetaMask/mobile-devs #(should be all tech leads from every lane)
```
Comment (sethkfman) [inline .github/CODEOWNERS:4]: Files specified below will override the rules here

**Pull request closed without merging**